CONTRIBUTING.txt

```diff
@@ -170,6 +170,48 @@ Once you've fixed all merge conflicts, do::
    <https://scikit-image.org/docs/dev/gitwash/development_workflow.html#rebasing-on-trunk>`__,
    but we squash and merge most PRs either way.
 
+Build environment setup
+-----------------------
+
+Development for scikit-image is possible on Windows, Linux, and MacOSX.
+Once you've cloned your fork of the git repository as detailed above,
+you should setup a Python development environment tailored for scikit-image.
+
+pip+virtualenv
+==============
+If you choose to use pip and virtualenv, then you may find the following bash
+commands::
+
+  virtualenv env
+  source env/bin/activate
+  # Install all development and runtime dependencies of scikit-image
+  pip install -r <(cat requirements/*.txt)
+
+conda
+=====
+
+If you prefer to use conda (Anaconda or Miniconda) for Python environment
+management, then you may find the following bash commands useful::
+
+  conda create --name skdev python=3.6
+  conda activate skdev
+  # Install all development and runtime dependencies of scikit-image
+  conda install `for i in requirements/{default,build,test,docs}.txt; do echo -n " --file $i "; done`
+
+You can choose a different name or a different Python version,
+depending on your preferences.
+If you are using the conda-forge channel and NumPy seems to be changing
```

Comment:
Removed, people can searech on conda-forge to find out how to solve this issue themselves.

Reply:
The other PR to remove requirements-parser was to ensure that it could build from anaconda defaults as well. So we are good on that front :D.

The documentation dependencies aren't available on anaconda defaults.